app/Providers/RouteServiceProvider.php

```diff
@@ -28,6 +29,12 @@ class RouteServiceProvider extends ServiceProvider
      */
     public const HOME = 'horizon';
 
+    public function boot()
+    {
+        parent::boot();
+        Route::model('version', Version::class);
```

Comment:
Curious to see you needing this - substitute bindings should make this redundant

Reply:
Added it when I was messing around trying to get it to work - removed!